src/sentry/db/models/fields/node.py

```diff
@@ -55,6 +55,7 @@ def __init__(self, field, id, data=None):
         #  in the case of something changing on the data model)
         self.ref_version = None
         self._node_data = data
+        self._data_size = data_size
```

Comment:
Remove this bs, this is just the length of the key that points into nodestore.

Reading the length from nodestore is trickier